.github/workflows/integration-tests.yml

```diff
@@ -26,7 +26,7 @@ jobs:
             github.com/ChainSafe/gossamer/dot,
             github.com/ChainSafe/gossamer/dot/core,
             github.com/ChainSafe/gossamer/dot/rpc/modules,
-            github.com/ChainSafe/gossamer/lib/babe,
+            github.com/ChainSafe/gossamer/lib/babe
```

Comment:
Is this necessary? :thinking: A bit OOS too. But sure, why not as well :smile:

Reply:
:joy: didn't mean to do this.  I'll remove it